README.md

```diff
@@ -133,16 +136,16 @@ Want your instance added here? Make a GitHub issue or make a PR! (this is for ge
 
 [phtn.app](https://phtn.app) is the official instance and will get updates instantly.
 
-
-| Instance                                                                             | Location    | Contact                                                                            |
-| --------------------------------------------------------------------------------------- | ------------ | ------------------------------------------------------------------------------------- |
-| [phtn.app (Official)](https://phtn.app)                                                 | 🇺🇸 US West   | [photon@xylight.dev](mailto:photon@xylight.dev)                                       |
+| Instance                                                                                | Location       | Contact                                                                               |
+|-----------------------------------------------------------------------------------------|----------------|---------------------------------------------------------------------------------------|
+| [phtn.app (Official)](https://phtn.app)                                                 | 🇺🇸 US West   | [photon@xylight.dev](mailto:photon@xylight.dev)                                       | 
+| [lemdro.id](https://lemdro.id)                                                          | Global         | [support@lemdro.id](mailto:support@lemdro.id)                                         |
```

Comment:
`lemdro.id` doesn't allow setting custom instance and README.md states that "this is for general purpose Photon instances". It should be excluded.